pkgs/development/tools/udis86/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, autoreconfHook, python }:
+
+stdenv.mkDerivation rec {
+  pname = "udis86";
+  version = "1.7.2";
+
+  src = fetchurl {
+    url = https://github.com/vmt/udis86/archive/v1.7.2.tar.gz;
+    sha256 = "0yappb8dxwsq6jcqm4zq51iv6g53nzsx7czznp2l728n29z7ymj3";
+  };
+
```

Comment:
to keep the closure small on downstream packages, you can use `outputs`
```suggestion
  outputs = [ "bin" "out" "dev" "lib" ];
```
this way, if someone just needs `udcli`, then can just say `udis86` (since it's the first ouput) or `udis86.bin`. If they just needs the headers, then `udis86.dev`, the libs go to `udis86.lib`, and the docs go to `udis86.out`.

This package is relatively small, so the impact isn't huge.

```
[nix-shell:~/.cache/nix-review/pr-66099-1]$ nix-store -q --tree ./result
/nix/store/qzbjy7cyjc2bjzwg0chrqz1lv19z1qb3-udis86-1.7.2

[nix-shell:~/.cache/nix-review/pr-66099-1]$ nix path-info -Sh ./result/
/nix/store/qzbjy7cyjc2bjzwg0chrqz1lv19z1qb3-udis86-1.7.2         222.4K

[nix-shell:~/.cache/nix-review/pr-66099-1]$ nix path-info -Sh ./result-bin
/nix/store/69rvyxjds60gf97k4i4j51jnnvg3iiz3-udis86-1.7.2-bin      27.3M

[nix-shell:~/.cache/nix-review/pr-66099-1]$ nix path-info -Sh ./result-lib
/nix/store/hzd10lz00sq7h97p92zx7abik4b0r7n7-udis86-1.7.2-lib      27.1M

[nix-shell:~/.cache/nix-review/pr-66099-1]$ nix path-info -Sh ./result-dev
/nix/store/9a0bnyvwkxsgxsdncnirmk3p65r7y1xy-udis86-1.7.2-dev      27.3M

[nix-shell:~/.cache/nix-review/pr-66099-1]$ nix-store -q --tree ./result
/nix/store/qzbjy7cyjc2bjzwg0chrqz1lv19z1qb3-udis86-1.7.2

[nix-shell:~/.cache/nix-review/pr-66099-1]$ nix-store -q --tree ./result-bin
/nix/store/69rvyxjds60gf97k4i4j51jnnvg3iiz3-udis86-1.7.2-bin
+---/nix/store/iykxb0bmfjmi7s53kfg6pjbfpd8jmza6-glibc-2.27
|   +---/nix/store/iykxb0bmfjmi7s53kfg6pjbfpd8jmza6-glibc-2.27 [...]
+---/nix/store/7sjhjsmjb92ahf3i339rhysk641chfvi-udis86-1.7.2
    +---/nix/store/iykxb0bmfjmi7s53kfg6pjbfpd8jmza6-glibc-2.27 [...]
    +---/nix/store/7sjhjsmjb92ahf3i339rhysk641chfvi-udis86-1.7.2 [...]

[nix-shell:~/.cache/nix-review/pr-66099-1]$ nix-store -q --tree ./result-lib
/nix/store/hzd10lz00sq7h97p92zx7abik4b0r7n7-udis86-1.7.2-lib
+---/nix/store/iykxb0bmfjmi7s53kfg6pjbfpd8jmza6-glibc-2.27
|   +---/nix/store/iykxb0bmfjmi7s53kfg6pjbfpd8jmza6-glibc-2.27 [...]
+---/nix/store/hzd10lz00sq7h97p92zx7abik4b0r7n7-udis86-1.7.2-lib [...]

[nix-shell:~/.cache/nix-review/pr-66099-1]$ nix-store -q --tree ./result-dev
/nix/store/9a0bnyvwkxsgxsdncnirmk3p65r7y1xy-udis86-1.7.2-dev
+---/nix/store/7sjhjsmjb92ahf3i339rhysk641chfvi-udis86-1.7.2
|   +---/nix/store/iykxb0bmfjmi7s53kfg6pjbfpd8jmza6-glibc-2.27
|   |   +---/nix/store/iykxb0bmfjmi7s53kfg6pjbfpd8jmza6-glibc-2.27 [...]
|   +---/nix/store/7sjhjsmjb92ahf3i339rhysk641chfvi-udis86-1.7.2 [...]
+---/nix/store/69rvyxjds60gf97k4i4j51jnnvg3iiz3-udis86-1.7.2-bin
    +---/nix/store/iykxb0bmfjmi7s53kfg6pjbfpd8jmza6-glibc-2.27 [...]
    +---/nix/store/7sjhjsmjb92ahf3i339rhysk641chfvi-udis86-1.7.2 [...]
```